lib/api/client-commands/network/setConditions.js

```diff
@@ -1,13 +1,13 @@
-const ClientCommand = require('./_base-command.js');
-const {Logger} = require('../../utils');
+const ClientCommand = require('../_base-command.js');
+const {Logger} = require('../../../utils');
 
 /**
  * 
  * Command to set Chrome network emulation settings.
  *
  * @example
  *  this.demoTest = function (browser) {
- *    browser.setNetworkConditions({
+ *    browser.network.setConditions({
  *      offline: false,
  *      latency: 50000,
  *      download_throughput: 450 * 1024,
```

Comment:
Can you change this example and the one in `indes.d.ts` file to use the `describe` syntax?